build.gradle.kts

```diff
@@ -52,6 +52,14 @@ dependencies {
     testImplementation("org.jetbrains:annotations:${Versions.jetbrainsAnnotations}")
 }
 
+configurations.configureEach {
+    resolutionStrategy.eachDependency {
+        if (requested.group == "org.jetbrains.kotlin" && requested.name.startsWith("kotlin")) {
+            useVersion("1.4.32")
+        }
+    }
+}
```

Comment:
Depepndency diff caused by this change should be visible in the comment:
![image](https://user-images.githubusercontent.com/36954793/116827019-44f12d80-ab97-11eb-9337-bd00175c7068.png)

Related blogpost: https://developer.squareup.com/blog/surfacing-hidden-change-to-pull-requests/